src/lib/import-export/import/importers/importer.ts

```diff
@@ -83,14 +138,17 @@ export class DefaultImporter extends EventEmitter implements Importer {
 				console.error( `Error processing ${ sqlFile }:`, error );
 				throw error;
 			} finally {
-				try {
-					await fsPromises.unlink( tmpPath );
-				} catch ( unlinkError ) {
-					console.error( `Failed to delete temporary file ${ tmpPath }:`, unlinkError );
-				}
+				await this.safelyDeleteFile( tmpPath );
 			}
 		}
-		this.emit( ImportEvents.IMPORT_DATABASE_COMPLETE );
+	}
+
+	protected async safelyDeleteFile( filePath: string ): Promise< void > {
+		try {
+			await fsPromises.unlink( filePath );
+		} catch ( error ) {
+			console.error( `Failed to delete temporary file ${ filePath }:`, error );
```

Comment:
From the function's name, I understand this function is used to remove any file. However, this error message specifies a temporary file. We could either update the message or rename the function to make it consistent.

Reply:
Updated! https://github.com/Automattic/studio/pull/424/commits/12d059f2e3796182cd9aed398ae0af4b21251fde
Thanks for the advice! 👌